app/src/main/java/com/google/droidjump/leveldata/InfiniteLevel.java

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.droidjump.leveldata;
+
+import android.content.res.Resources;
+import android.util.Log;
+import org.json.JSONException;
+import org.json.JSONObject;
+
+public class InfiniteLevel implements LevelStrategy {
+
+    final static String baseSpeedKey = "baseSpeed";
```

Comment:
nit:for final and static variables use capital letters.

BASE_SPEED_KEY = baseSpeed

Reply:
Done